app_dart/lib/src/service/datastore.dart

```diff
@@ -43,10 +43,13 @@ class DatastoreService {
   /// The [limit] argument specifies the maximum number of commits to retrieve.
   ///
   /// The returned commits will be ordered by most recent [Commit.timestamp].
-  Stream<Commit> queryRecentCommits({int limit = 100, int timestamp}) {
+  Stream<Commit> queryRecentCommits(
+      {int limit = 100, int timestamp, String branch}) {
     timestamp ??= DateTime.now().millisecondsSinceEpoch;
+    branch ??= 'master';
     final Query<Commit> query = db.query<Commit>()
       ..limit(limit)
+      ..filter('branch =', branch)
```

Comment:
Did we back-fill existing data to add 'master' as the branch?

Reply:
I had written a debug handler for updating data at one point for this purpose, but never checked it in for security reasons...

As you say, it's lower priority if the affected rows are old enough to not show on the dashboard by default.